### ISSUE TITLE
EventBase does not handle MultiInput Webhook Event Format (Throws JsonReaderException)

### ISSUE THREAD

**Issue by DanielStout5** [Nexmo/nexmo-dotnet#209]
If you try to use the `Input` class from your library, it says it is obsolete:

`
This item has been rendered obsolete due to the new multi-input functionality. Please add dtmf arguments to your input action and use the MultiInput object - see: https://developer.nexmo.com/voice/voice-api/ncco-reference#dtmf-input-settings
`

If you call `EventBase.ParseEvent(json);` where `json` contains the new dtmf format, it will throw an exception:

`
Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: {. Path 'dtmf', line 1, position 53.
`

The culprit is this code in EventBase.cs:
```
            else if (dtmfProperty != null)
            {
                return JsonConvert.DeserializeObject<Input>(json, settings);
            }
```

That should be updated to deserialize as `Input` if `dtmfProperty` is a string, or `MultiInput` if it's an object.

**Comment by slorello89** [comment]
Hi @DanielStout5 - thanks so much for pointing this out. I've opened #210 to address this issue. FYI - I have another PR opened that will help you out with parsing your webhooks generally that should be released at the same time #208. This will allow you to parse straight out of the body/query of your asp.net/asp.net core API controllers.

Thanks for your input!

**Comment by slorello89** [comment]
Hi @DanielStout5 - this issue is resolved by the release of v5.2.0 so I'm going to close this issue.
